bkmark-generator.py

```diff
@@ -0,0 +1,61 @@
+##
```

Comment:
I'd like to have the `*.html` and `*.py` files separated from the current rootdir into a new folder named "bookmarks". Would that be possible?